apps/dashboard/app/helpers/batch_connect/sessions_helper.rb

```diff
@@ -82,18 +82,24 @@ def time(session)
   end
 
   def host(session)
-    content_tag(:p) do
-      concat content_tag(:strong, t('dashboard.batch_connect_sessions_stats_host'))
-      concat " "
-      concat(
-        session.connect.host ? link_to(
-          session.connect.host, OodAppkit.shell.url(
-            host: session.connect.host).to_s,
-            target: "_blank",
-            class: "btn btn-primary btn-sm fas fa-terminal"
-          ) : t('dashboard.batch_connect_sessions_stats_undetermined_host')
-      )
-    end if session.running?
+    if ::Configuration.ood_bc_ssh_to_compute_node
+      content_tag(:p) do
+        concat content_tag(:strong, t('dashboard.batch_connect_sessions_stats_host'))
+        concat " "
+        concat(
+          session.connect.host ? link_to(
+            session.connect.host, OodAppkit.shell.url(
+              host: session.connect.host).to_s,
+              target: "_blank",
+              class: "btn btn-primary btn-sm fas fa-terminal"
+            ) : t('dashboard.batch_connect_sessions_stats_undetermined_host')
+        )
+      end if session.running?
+    else
+      content_tag(:p) do
```

Comment:
This should be: https://github.com/OSC/ood-dashboard/blob/00667cc845fb35bcc0771ab018bf07192571a7fa/app/helpers/batch_connect/sessions_helper.rb#L84-L90